docs/releases/v2023.2.rst

```diff
@@ -0,0 +1,278 @@
+Gluon 2023.2
+============
+
+Important notes
+---------------
+
+Upgrades to v2023.2 and later releases are only supported from releases v2022.1 and later.
+This is due to migrations that have been removed to simplify maintenance.
+
+
+Deprecation of Tunneldigger VPN
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Tunneldigger is set to be removed from the Gluon base repository in the next major Gluon release. It is recommended
+to migrate to fastd or WireGuard. Tunneldigger will be moved to the
+community-packages repository and can be installed from there as an alternative.
+
+
+Site changes
+------------
+
+Image customization
+~~~~~~~~~~~~~~~~~~~
+
+`GLUON_FEATURES` as well as `GLUON_PACKAGES` have been replaced by a more flexible customization framework
+based on Lua. Package and feature selection can be specified more granularly at both target and device level.
+
+Additionally, this framework also allows communities to specify which devices should or should not be built.
+For more information, see the :ref:`documentation <site-image-customization>`.
```

Comment:
Should we also add a short example of old vs. new configuration directly to the release notes, like we did with other site changes in the past?

Reply:
I think the reference to the documentation is enough. I was thinking about referencing community sites that already switched (like ffda and muc) to show practical examples. I see them as more valuable. What do you think?

Reply:
I'd still prefer a simple before/after comparison of two examples in the release notes, and I've opened https://github.com/freifunk-gluon/gluon/pull/3127 to improve the referenced docs. Should I just push suggested improvements as new commits to this branch, if adding them as comments is inconvenient?